spikeinterface/widgets/legacy_mpl_widgets/gtstudy.py

```diff
@@ -121,9 +121,10 @@ def plot(self):
             ax.bar(x, m[col].values, yerr=yerr, width=1/(ncol+2), color=cmap(c), label=clean_labels[c])
 
         ax.legend()
+        ax.set_yscale('log')
```

Comment:
Can we make this optional? I would have an additional arg: `log_scale` False by default. What do you think?

Reply:
We could, but with your whole widgets approach, Sam told me not to have any arguments in the plot() function. But I agree that this should be optional

Reply:
This is now optional, with the logscale argument (default is False)